.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,110 @@
+name: Release workflow
+# This workflow is triggered on creating tags to master or a opendistro release branch
+on:
+  push:
+  tags:
+    -'v*'
+    branches:
+      - master
+      - opendistro-*
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        java: [12]
+    # Job name
+    name: Build Alerting with JDK ${{ matrix.java }}
+    # This job runs on Linux
+    runs-on: ubuntu-latest
+    steps:
+      # This step uses the checkout Github action: https://github.com/actions/checkout
+      - name: Checkout Branch
+        uses: actions/checkout@v1
```

Comment:
Might be a good idea to change this to ```checkout@v2```. There is a known issue where reruns using this GitHub action [end up failing](https://github.com/actions/checkout/issues/23) and I believe it was fixed in ```v2```.